### PR TITLE
Embed no function

### PR DESCRIPTION
**Pull request by SylvainCorlay**
@jdfreder another take on the how to embed widgets, which is to use a `text/json` mime type.

The problem I see is that I must use windows.onload.

### CODE REVIEW

File: jupyter-js-widgets/src/embed-webpack.js [hunk 1]
```diff
@@ -7,13 +7,12 @@
 // Load jquery and jquery-ui
 var $ = require('jquery');
 require('jquery-ui');
-$.getQueryParameters = window.$.getQueryParameters;
```
Comment (SylvainCorlay) [inline jupyter-js-widgets/src/embed-webpack.js]: I don't  understand why you were refering to `window.$`at this point.

Comment (jdfreder) [inline jupyter-js-widgets/src/embed-webpack.js]: These lines are left over from the docs, without these lines the docs will break.  However, I don't think these lines should exist here, instead the fix belongs in the docs.

Comment (SylvainCorlay) [inline jupyter-js-widgets/src/embed-webpack.js]: but how is `window.$` defined at this point?

Comment (jdfreder) [inline jupyter-js-widgets/src/embed-webpack.js]: By the jquery in the theme

Comment (jdfreder) [inline jupyter-js-widgets/src/embed-webpack.js]: Ithink the real problem here is that the jquery version of the doc theme !== the version specified in the package.json

Comment (SylvainCorlay) [inline jupyter-js-widgets/src/embed-webpack.js]: ok, so indeed, it should be moved to the docs thing. You are right.

File: jupyter-js-widgets/src/embed-webpack.js [hunk 2]
```diff
@@ -21,37 +20,25 @@ console.info('jupyter-js-widgets loaded successfully');
...
 // Magic global widget rendering function:
-function renderWidgetState(widgetStateObject) {
-
-  // Create the container div and insert it into the DOM immediately above the
-  // current script tag.
-  console.info('Inserting widget(s)...');
-  var container = document.createElement('div');
-  container.innerHTML = `
-    <div class="widget-loading">
-      <span class="glyphicon glyphicon-hourglass glyphicon-spin" aria-hidden="true"></span>
```
Comment (jdfreder) [inline jupyter-js-widgets/src/embed-webpack.js]: Here you remove the loading animation, you should also remove the `glyphicon-spin` class from the css which I had to add manually.

Comment (jdfreder) [inline jupyter-js-widgets/src/embed-webpack.js]: We can address this in another PR

**Pull request merged**